app/src/main/java/tech/ula/MainActivity.kt

```diff
@@ -112,6 +113,10 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (item.itemId == R.id.terms_and_conditions) {
+            val intent = Intent("android.intent.action.VIEW", Uri.parse("https://userland.tech/eula"))
```

Comment:
What do you think of creating an XML file that holds static URLs?  I was thinking this could be abstracted in a way similar to how Strings.xml holds static strings.  Maybe the URLs could even be placed inside strings.xml

Reply:
I think it's adds a layer of unnecessary complexity. Off the top of my head, I can't think of any URLs that are reused by different pieces of code yet. Additionally, it would mean passing a `Resource` object to each method/class that needed to access the URLs.

Reply:
I was thinking that there's some benefit of being able to look at one location for all strings related to URLs.  I suppose it's not necessary yet since URLs aren't being reused around the codebase.